docs/data/charts/axis/axis.md

```diff
@@ -0,0 +1,101 @@
+---
+product: charts
+title: Charts - Axis
+---
+
+# Charts - Axis
+
+<p class="description">Axis provides associate values to element positions</p>
+
+Axis are used in the following charts: `<LineChart />`, `<BarChart />`, `<ScatterChart />`.
+
+## Defining axis
+
+Like your data, axis definition play a central role in the charts rendering.
+It's responsible of the mapping between your data and elements positions.
+
+You can define custom axis by using `xAxis` and `yAxis` props.
```

Comment:
I start wondering if it should be call `xAxes` because it's an array of axes. But it feels weird because I do not remember to have use "axes" in libraries I used for charting

Reply:
Yeah, the prop name in plural form does indeed sound strange. If you could investigate what plural form are other libraries using that would be best 👌 🤔